openshift/sidecar-conf/fluent-bit.conf

```diff
@@ -0,0 +1,59 @@
+[SERVICE]
+  Flush        5
+  Daemon       Off
+  # define the log format (see additional config map key/value)
+  Parsers_File  parsers.conf
+  Log_Level    warn
+  HTTP_Server   On
+  HTTP_Listen   0.0.0.0
+  HTTP_Port     2020
+
+[INPUT]
+  # get logs from file written by node app (eg: CDOGS)
+  Name   tail
+  Path   /var/log/*
+  Tag    app
+  Parser apache
+
+[FILTER]
+  # exclude kube probe logs from app logs
+  name   grep
+  match  app
+  Exclude  agent kube*
+
+[FILTER]
+  # modify log entry to include more key/value pairs
+  name    record_modifier
+  match   app
+  # add pod name
+  Record  hostname ${HOSTNAME}
+  # add product name (eg: 'ches')
+  Record  product ches
+  # add namespace
+  Record namespace 9f0fbe-dev
```

Comment:
This might work for dev, but if this gets merged into master, this will not work for TEST and PROD (either make sure there's instructions to let us know to change this, or re-template this as another openshift template file with parameters)